app/controllers/register/trustees/individual/TrusteePhoneController.scala

```diff
@@ -17,36 +17,67 @@
 
 import config.FrontendAppConfig
-import controllers.Retrievals
+import controllers.PhoneNumberController
 import controllers.actions._
+import forms.PhoneFormProvider
+import identifiers.register.trustees.individual.{TrusteeNameId, TrusteePhoneId}
 import javax.inject.Inject
 import models.{Index, Mode}
 import navigators.Navigator
+import play.api.data.Form
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent}
 import services.UserAnswersService
-import uk.gov.hmrc.play.bootstrap.controller.FrontendController
-import utils.Enumerable
-import utils.annotations.TrusteesIndividual
+import viewmodels.{CommonFormWithHintViewModel, Message}
 
 import scala.concurrent.ExecutionContext
 
-class TrusteePhoneController @Inject()(val appConfig: FrontendAppConfig,
-                                       val messagesApi: MessagesApi,
-                                       val userAnswersService: UserAnswersService,
-                                       val navigator: Navigator,
-                                       authenticate: AuthAction,
-                                       getData: DataRetrievalAction,
-                                       allowAccess: AllowAccessActionProvider,
-                                       requireData: DataRequiredAction
-                                     )(implicit val ec: ExecutionContext) extends FrontendController with Retrievals with I18nSupport with Enumerable.Implicits {
+class TrusteePhoneController @Inject()(
+                                        val appConfig: FrontendAppConfig,
+                                        override val messagesApi: MessagesApi,
+                                        authenticate: AuthAction,
+                                        getData: DataRetrievalAction,
+                                        override val userAnswersService: UserAnswersService,
+                                        allowAccess: AllowAccessActionProvider,
+                                        requireData: DataRequiredAction,
+                                        val navigator: Navigator,
+                                        formProvider: PhoneFormProvider
+                                      )(implicit val ec: ExecutionContext) extends PhoneNumberController with I18nSupport {
+
+
+  protected val form: Form[String] = formProvider()
+
+  private def viewModel(mode: Mode, srn: Option[String], index: Index): Retrieval[CommonFormWithHintViewModel] =
+    Retrieval {
+      implicit request =>
+        TrusteeNameId(index).retrieve.right.map {
+          details =>
+            CommonFormWithHintViewModel(
+              routes.TrusteePhoneController.onSubmit(mode, index, srn),
+              Message("messages__common_phone__heading", Message("messages__common__address_years__trustee").resolve),
+              Message("messages__common_phone__heading", details.fullName),
+              Some(Message("messages__establisher_phone__hint")),
```

Comment:
please rename this message as it says "messages__establisher_phone__hint" but it is applied for trustee

Reply:
Done. Also changed email hint